cicd/build.sh

```diff
@@ -200,8 +200,8 @@ if [ "$DEV" == true ]; then
     install_py_req req_dev.txt
 
     # Solving numpy libgfortran-ed201abd.so.3.0.0 dependency problem
-    pip uninstall -y numpy
-    pip install numpy --no-binary :all:
+    #pip uninstall -y numpy
```

Comment:
This is not required as we have moved the code outside of if statement. I have commented the code earlier, please remove it.

Reply:
Done with changes